docusaurus/docs/getting-started.mdx

```diff
@@ -22,44 +24,54 @@ Plugin tools consists of two packages:
   queryString="current-package-manager"
 />
 
-Follow the prompts to scaffold a Grafana plugin. [See here](./creating-a-plugin.mdx) for detailed information about creating a plugin.
+For next steps, see [Creating a plugin](./creating-a-plugin.mdx) 
 
 :::note
 
-Plugins previously scaffolded with `@grafana/toolkit` can be migrated. Please see [this migration guide](./migrating-from-toolkit).
+If you have previously built a plugin with `@grafana/toolkit`, you can use our plugin tools to simplify migration. For more information, refer to the [Migration guide](./migrating-from-toolkit).
 
 :::
 
-### What you'll need
+### Before you begin 
 
-- [Go](https://go.dev/doc/install) version 1.18 or above
-- [Mage](https://magefile.org/)
-- [Node.js](https://nodejs.org/en/download/) version 16 or above:
-  - When installing Node.js, you are recommended to check all checkboxes related to dependencies.
-- Optionally [Yarn 1](https://classic.yarnpkg.com/lang/en/docs/install) or [PNPM](https://pnpm.io/installation)
-- [Docker](https://docs.docker.com/get-docker/)
+#### Supported operating systems
 
-### Supported Operating Systems
-Create Plugin currently supports the following operating systems:
+Grafana plugin tools work with the following operating systems:
 
 - Linux
 - macOS
 - Windows 10+ with WSL (Windows Subsystem for Linux)
 
-### Choosing a package manager
-Starting from v1.3 of Create Plugin when you create a new plugin, the CLI will use `npm`, `pnpm` or `yarn` depending on which tool you use to run `@grafana/create-plugin`.
+#### Supported Grafana version
+
+We recommend that you build for a version of Grafana later than v9.0. The plugin tools may work with earlier versions, but Grafana Labs may not provide support for them.
+
+#### Recommended tooling
+
+You'll need to have the following tools set up:
+
+- [Go](https://go.dev/doc/install) version 1.18 or later.
+- [Mage](https://magefile.org/).
+- [Node.js](https://nodejs.org/en/download/) version 16 or later.
+  - We recommend that you should install Node.js with all checkboxes related to dependencies checked.
+- [Docker](https://docs.docker.com/get-docker/).
+- Optional: [Yarn 1 (Classic)](https://classic.yarnpkg.com/lang/en/docs/install) or [pnpm](https://pnpm.io/installation).
+
+#### Choose a package manager
+
+When you first run `@grafana/create-plugin`, choose your package manager `npm`, `pnpm`, or `yarn`.
```

Comment:
I'm not sure this change makes it clearer that create-plugin features support for the three package managers mentioned. A developer will choose the package manager when running the command and based on the chosen package manager the output is different. I don't really know how to word it any better though...

`@grafana/create-plugin` supports `npm`, `pnpm` and `yarn`. Scaffolded plugins will inherit the package manager you use when scaffolding.

Or maybe...

When creating a new plugin, the CLI will use `npm`, `pnpm` or `yarn` as the plugins package manager, depending on which tool you use to run `@grafana/create-plugin`.

Reply:
@jackw - Regarding the docs in the plugin-tools repo, Timur, David, and I have planned to move the docs to live with the other docs. When this is done, or when a developer portal emerges, the docs at plugin-tools would be deprecated. Conversations on this are ongoing.